Datastore/src/Query/AggregationQueryResult.php

```diff
@@ -162,12 +174,46 @@ public function getTransaction()
     }
 
     /**
-     * Gets the timsstamp when Aggregation was executed.
+     * Gets the timestamp when Aggregation was executed.
      *
      * @return Timestamp
      */
     public function getReadTime()
     {
         return $this->readTime;
     }
+
+    /**
+     * Gets the ExplainMetrics object if the explainOptions parameter was set for
+     * the query
+     *
+     * @return null|ExplainMetrics
+     */
+    public function getExplainMetrics(): null|ExplainMetrics
+    {
+        return $this->explainMetrics;
+    }
+
+    private function fixDurationFormat(array $metrics): array
+    {
+        // The current protobuf library does not support the current json representation
+        // of the well-known type Duration.
```

Comment:
This doesn't make sense to me, since the current protobuf library is where the Duration type is defined.

Reply:
For documentation purposes:
This got moved and created a custom serializer inside the GrpcTrait.php file in Core.